profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.multilingual.inc

```diff
@@ -0,0 +1,701 @@
+        'field' => 'field_newsroom_topic_id',
+      ),
+    ),
+    'field_newsroom_ec_content' => array(
+      'query' => 'infsonewsroom:ContextECContent/text()',
+    ),
+    'field_newsroom_item_venue' => array(
+      'query' => 'infsonewsroom:ContextVenue',
+    ),
+    'field_newsroom_organizer' => array(
+      'query' => 'infsonewsroom:ContextOrganiser',
+    ),
+    'field_newsroom_author' => array(
+      'query' => 'infsonewsroom:ContextAuthor',
+    ),
+    'field_eur_lex_link:url' => array(
+      'query' => 'infsonewsroom:ContextEurlex',
+    ),
+    'field_ted_link:url' => array(
+      'query' => 'infsonewsroom:ContextTed',
+    ),
+    'field_newsroom_registration_link:url' => array(
+      'query' => 'infsonewsroom:ContextRegistrationLinkText',
+    ),
+    'field_newsroom_registration_link:title' => array(
+      'query' => 'infsonewsroom:ContextRegistrationLink',
+    ),
+    'body' => array(
+      'query' => 'infsonewsroom:FullContent',
+    ),
+    'field_newsroom_item_contact_text' => array(
+      'query' => 'infsonewsroom:ContextContactText',
+    ),
+    'field_newsroom_item_contact_mail' => array(
+      'query' => 'infsonewsroom:ContextContactEmail',
+    ),
+    'field_newsroom_item_linked_obj' => array(
+      'query' => 'infsonewsroom:FullLinkedObject',
+    ),
+    'field_newsroom_item_pre_title' => array(
+      'query' => 'infsonewsroom:FullPreTitle',
+    ),
+    'field_newsroom_item_quote_box' => array(
+      'query' => 'infsonewsroom:FullQuoteBox',
+    ),
+    'field_newsroom_item_short_title' => array(
+      'query' => 'infsonewsroom:BasicShortTitle',
+    ),
+    'field_newsroom_item_speaker' => array(
+      'query' => 'infsonewsroom:ContextSpeaker',
+    ),
+    'field_teaser' => array(
+      'query' => 'infsonewsroom:BasicTeaser',
+    ),
+    '_feed_changed' => array(
+      'query' => 'infsonewsroom:pubUpdateDate/text()',
+      'tamper' => array(
+        'plugin_id' => 'strtotime',
+      ),
+    ),
+  );
+
+  $premapping = _nexteuropa_newsroom_check_translations($premapping);
+  $importer_values = _nexteuropa_newsroom_prepare_dynamic_mapping($premapping, NEXTEUROPA_NEWSROOM_ITEM_IMPORTER);
+  $importer_values['mapping'][] = array(
+    'source' => 'Blank source 1',
+    'target' => 'language',
+    'unique' => FALSE,
+  );
+
+  $feeds_importer = _nexteuropa_newsroom_build_importer(NEXTEUROPA_NEWSROOM_ITEM_IMPORTER, 'Newsroom Items Multilanguage', 'Imports Newsroom Items from the Newsroom service', $importer_values, NEXTEUROPA_NEWSROOM_CONTENT_TYPE, 'node');
+
+  // Create the tamper for the language.
+  _nexteuropa_newsroom_dynamic_tamper(NULL);
+  // Import the importer.
+  _nexteuropa_newsroom_import_importer($feeds_importer);
+}
+
+/**
+ * Build importer class.
+ */
+function _nexteuropa_newsroom_build_importer($id, $title, $description, $importer_values, $bundle, $type = 'node', $rss_context = '//channel/item', $import_period = 3600) {
+  $feeds_importer = new stdClass();
```

Comment:
You should somehow set the limit for the feeds importer(s) because it can cause performance issues if you just wait for the end of the current job. Also possible to put it to cron and makes the whole process cron controlled, in this case needs to set the process_in_background. More details are here e.g.: http://nielsdefeyter.nl/archive/2013/setup-large-imports-drupal-feeds